css/pico.amber.css

```diff
@@ -1108,7 +1103,6 @@ td {
   background-color: var(--pico-background-color);
   color: var(--pico-color);
   font-weight: var(--pico-font-weight);
-  text-align: left;
```

Comment:
`left` and `start` are identical so they are not needed at the current time when IE doesn't exist.